### PR TITLE
Multiple ski resorts

### PR DESCRIPTION
**Pull request by Mafusn**
#37 
- Fly to multiple ski areas
- Fetches pistes and liftes after new skiAreaId is selected
- Possible to place the A marker after switching ski area
- "Generate route" and "Select ski area" buttons are same size
- Tests

### CODE REVIEW

**Review by Jatewo — CHANGES_REQUESTED** [review]
### Small changes

Could use some testing in cypress (just to ensure that the functionality is not broken later on)

Other than that a little change is requested (see the comment)

File: src/components/Map/FeatureHandler.jsx [hunk 1]
```diff
@@ -18,8 +18,8 @@ export default function FeatureHandler(props) {
 			{props.lifts && <GeoJSON data={props.lifts} style={setLiftStyle} />}
 
 			{/* Bigger weight to make the pistes and lifts easier to click */}
-			{props.pistes && <GeoJSON data={props.pistes} style={{ ...setPisteColor, color: 'transparent', weight: 15 }} onEachFeature={addPisteDetails} />}
-			{props.lifts && <GeoJSON data={props.lifts} style={{ ...setLiftStyle, color: 'transparent', weight: 15 }} onEachFeature={addLiftDetails} />}
+			{props.pistes && <GeoJSON key={`pistes-${props.key}`} data={props.pistes} style={{ ...setPisteColor, color: 'transparent', weight: 15 }} onEachFeature={addPisteDetails} />}
```
Comment (Jatewo) [inline src/components/Map/FeatureHandler.jsx:21]: These keys should be removed. `props.key` will always return `undefined` and the key you need for leaflet to update can be defined in the FeatureHandler component in SkiMap.jsx only with the desired effect.

File: src/components/Map/FeatureHandler.jsx [hunk 2]
```diff
@@ -18,8 +18,8 @@ export default function FeatureHandler(props) {
...
+			{props.lifts && <GeoJSON key={`lifts-${props.key}`}data={props.lifts} style={{ ...setLiftStyle, color: 'transparent', weight: 15 }} onEachFeature={addLiftDetails} />}
```
Comment (Jatewo) [inline src/components/Map/FeatureHandler.jsx:22]: See above

File: src/components/Map/FeatureHandler.jsx [hunk 3]
```diff
@@ -28,4 +28,5 @@ FeatureHandler.propTypes = {
 	route: PropTypes.object,
 	pistes: PropTypes.arrayOf(object),
 	lifts: PropTypes.arrayOf(object),
+	key: PropTypes.string
```
Comment (Jatewo) [inline src/components/Map/FeatureHandler.jsx:31]: See above

**Review by MortenMunk — CHANGES_REQUESTED** [review]
Please check comments by jatewo

**Review by Jatewo — APPROVED** [review]
LGTM 🥇

**Review by Jatewo — APPROVED** [review]

**Review by MortenMunk — APPROVED** [review]
LGTM

**Pull request merged**